templates/Default/engine/Default/includes/AllianceRankings.inc

```diff
@@ -1,12 +1,14 @@
 <div align="center">
-	<p>Here are the rankings of alliances by their <?php echo strtolower($RankingStat); ?>.</p>
-	<p>You alliance is ranked <?php echo number_format($OurRank); ?> out of <?php echo number_format($TotalRanks); ?> alliances.</p>
-	<?php $this->includeTemplate('includes/AllianceRankingsList.inc', array('RankingStat' => $RankingStat, 'Rankings' => $Rankings)); ?>
+	<p>Here are the rankings of alliances by their <?php echo strtolower($RankingStat); ?>.</p><?php
+	if (isset($OurRank)) { ?>
+		<p>You alliance is ranked <?php echo number_format($OurRank); ?> out of <?php echo number_format($TotalRanks); ?> alliances.</p><?php
```

Comment:
Could you change `You` to `Your` as part of this

Reply:
Good catch! Fixed.